### PR TITLE
feat: initial release

### PR DESCRIPTION
**Pull request by brian-patrick-3**
There's a lot going on here, but the main things to focus on are the stories under the "Charts" category.

### CODE REVIEW

**Review by peterpham-kyndryl — COMMENTED** [review]
From a quick glance. 

1. There are a lot of defined but unused variables and commented code. If they're not needed, they should be cleaned up.
2. We should consider defining literal strings separately so that we can easily handle multi-language support in the future.

File: src/common/config/chartTypes/bar.js [hunk 1]
```diff
@@ -1,7 +1,85 @@
+import colorPalettes from '../colorPalettes';
+
+const BackgroundColor = getComputedStyle(
```
Comment (peterpham-kyndryl) [inline src/common/config/chartTypes/bar.js:3]: This for example is defined but never used.

Comment (brian-patrick-3) [reply src/common/config/chartTypes/bar.js:3]: Yeah been doing a lot of experimentation. Will do a cleanup pass before merge.

File: src/common/config/globalOptions.js [hunk 2]
```diff
@@ -11,3 +56,88 @@ const defaultConfig = (ctx) => {
...
+ */
+const handleLegendHover = (e, item, legend) => {
+  const DatasetIndex = item.datasetIndex || 0;
+  const Datasets = legend.chart.data.datasets;
+  const Dataset = Datasets[DatasetIndex];
+
+  if (Array.isArray(Dataset.backgroundColor)) {
+    Dataset.backgroundColor.forEach((color, index, colors) => {
+      colors[index] =
+        index === item.index || color.length === 9 ? color : color + '4D';
```
Comment (peterpham-kyndryl) [inline src/common/config/globalOptions.js:79]: I think it'd better to assign number 9 and such things a descriptive constant name so they won't be magic numbers.

**Review by CunningDesigns — APPROVED** [review]

**Review by CunningDesigns — APPROVED** [review]

**Review by CunningDesigns — APPROVED** [review]

**Pull request merged**